public-ui/dispatch.yaml

```diff
@@ -0,0 +1,7 @@
+dispatch:
+  - url: "*/v1/*"
```

Comment:
This seems slightly too broad, e.g. if there were a UI page which had a "/v1/" somewhere in the path. Can you restrict it to "/v1/*", or can you inclue the base uRL in here?

Reply:
added base url. thanks.